src/jit/inline.cpp

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "jitpch.h"
+#ifdef _MSC_VER
+#pragma hdrstop
+#endif
+
+static const char* InlineExplanations[] = 
+{
+    "unused initial",
+#define INLINE_OBSERVATION(name, type, description, impact, target) description,
+#include "inline.def"
+#undef INLINE_OBSERVATION
```

Comment:
same for "inline final". Otherwise, all users need to adjust accordingly, and might forget (and create extra clutter)